.k8s/prod/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-court-data-ui
+  namespace: laa-court-data-ui-prod
+spec:
+  replicas: 2
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 100%
+  selector:
+    matchLabels:
+      app: laa-court-data-ui
+  template:
+    metadata:
+      labels:
+        app: laa-court-data-ui
+    spec:
+      containers:
+        - name: laa-court-data-ui-app
+          imagePullPolicy: Always
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/laa-get-paid/laa-court-data-ui:set-me
+          ports:
+            - containerPort: 3000
+          readinessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 15
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              path: /ping.json
+              port: 3000
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+                - name: X-Forwarded-Ssl
+                  value: "on"
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          env:
+            - name: ENV
+              value: 'prod'
+            - name: RACK_ENV
+              value: 'production'
+            - name: RAILS_ENV
+              value: 'production'
+            - name: DOMAIN_URL
+              value: 'https://prod.view-court-data.service.justice.gov.uk'
```

Comment:
should be https://view-court-data.service.justice.gov.uk

Reply:
👍